### DISCUSSION TITLE
New PyPI org

### DISCUSSION THREAD

**Discussion by webknjaz** [aio-libs/.github#26] [Announcements]
I've made a new PyPI org — https://pypi.org/org/aio-libs/. I'm still figuring out what to set up there but I moved all the org project I had access to there. I've invited a few folks but if I missed someone — let me know. The invitations expire in 3 days.

cc @aio-libs/admins @mjpieters @Dreamsorcerer

**Comment by Dreamsorcerer** [comment]
Appears I didn't have an account (well, I seemed to have a saved password in my browser without a username...).
Have created a new account with the same username now: https://pypi.org/user/Dreamsorcerer/

**Reply by webknjaz** [reply]
Yeah, I figured when I couldn't find it. Invited.

**Reply by webknjaz** [reply]
@Dreamsorcerer plz enable 2FA on PyPI. It supports both TOTP and FIDO2/Webauthn. So I recommend setting up all of them. Even if you don't have a FIDO2 USB token (like Solo v2 or Yubikey), modern Android phones have a built-in compatible token that you could configure.

**Reply by webknjaz** [reply]
@Dreamsorcerer looks like you still have it disabled. Could you fix your 2FA?

**Comment by mjpieters** [comment]
I don't think I got an invite yet; no email in my inbox or my spam folder.

My pypi account is https://pypi.org/user/mj/

**Reply by webknjaz** [reply]
Oh, I didn't know your handle, since haven't seen you on any of the projects there. You should have an invitation in your inbox now.

**Reply by mjpieters** [reply]
Check, I'm in now. I can't transfer the two aio-lib projects I'm owner of however as I'm not a Manager.

**Reply by mjpieters** [reply]
Thanks for the role update; I've tranferred `aiosignal` and `frozenlist` into the org. I now don't have access to these project's settings now, I think you need to re-add me as a maintainer there.

**Reply by webknjaz** [reply]
Yeah, I noticed that behavior when moving other projects. I think it's worth mentioning it in the warehouse repo..

**Reply by mjpieters** [reply]
Thanks for fixing the ownership settings!

I later saw that there are two paths to move a project into an org:

1. [Specific project] -> Settings -> *Transfer project to an organization*
2. Org -> Projects -> (at the bottom of the page) *Add project to organization*, select an existing project.

I used option 1, it could be that option 2 has a different outcome.

**Reply by webknjaz** [reply]
> it could be that option 2 has a different outcome.

Nope, it behaves like this in both cases.

**Comment by webknjaz** [comment]
UPD: Some of you may get emails about the privileges changes (I'm not sure who exactly gets them, but some might) — this is because I'm reviewing accesses and making sure that the owner access is on the org, not individual users (keeping them listed as maintainers). And I'm also looking into removing the bot account from projects that still have it listed for some reason (but have already migrated to secretless publishing: https://github.com/orgs/aio-libs/discussions/25).
